.github/workflows/test.yml

```diff
@@ -64,12 +62,6 @@ jobs:
         echo "GITHUB_X_PR_BASE_SHA=${{ github.event.pull_request.base.sha }}" >> $GITHUB_ENV
         echo "GITHUB_X_PR_BASE_REF=${{ github.event.pull_request.base.ref }}" >> $GITHUB_ENV
 
-    - name: Create GCP Credential File from secret
```

Comment:
@AdamMagaluk managed to finally work around this in https://github.com/viamrobotics/goutils/commit/ea3a4a57328a5311c357e74e6ac9c67e3de8b06a . annoying but it works and reduces a cred